community/governance/index.md

```diff
@@ -5,4 +5,125 @@ sidebar:
+* Help new developers get started
+* Reproduce bugs and add detail to issues
+* Join a SIG, discuss community issues, and/or lead a SIG
+
+### Reviewers
+
+Duties
+
+* Review, provide meaningful feedback and LGTM PRs
+* Triage GitHub issues, community forum submissions
+* Provide general support in Slack
+
+Qualification guidelines
+
+* Known active participant in the community
+* Has submitted at least 5 PRs of significant scope that are merged
+* Sponsored by 2 Approvers
+
+[Current Reviewers](https://github.com/spinnaker/spinnaker/blob/master/reviewers.md)
+
+If you’d like to nominate yourself as a Reviewer, please reach out to the [TOC](#technical-oversight-committee).
+
+### Approvers
+
+Duties
+
+* Final review for PRs
+* Merge code into master
+* All Reviewer duties
+
+Qualification guidelines
+
+* Belongs to an organization with significant stake in project
+* Known active participant in the community
+* Has submitted at least 5 PRs of significant scope that are merged
+* Has reviewed 5 PRs of significant scope
+* Sponsored by 2 Approvers
+
+[Current Approvers](https://github.com/spinnaker/spinnaker/blob/master/approvers.md)
+
+If you’d like to nominate yourself as an Approver, please reach out to the [TOC](#technical-oversight-committee).
+
+### Technical Oversight Committee
+
+The TOC is responsible for the overall technical management of the project, ultimately managing the day-to-day running of the project.
+
+Duties
+
+* Set agenda for, facilitate, and drive consensus in the TOC meetings
+* Final escalation point for technical guidance
+* Review, approve and revoke appointments for Reviewers and Approvers
+* Drive technical management and any specific guidelines (e.g. code requirements/conventions, SLA on PR reviews)
+
+Qualifications
+
+* Nominated by community
+* Appointed by Steering Committee
+
+Logistics
+
+* [Agenda doc](https://docs.google.com/document/d/1PxIA1XE3nzqLykOFW-AqdU5u9F8cFh3jDvUA41P2cUM/edit)
+* Email: [toc@spinnaker.io](mailto:toc@spinnaker.io)
+
+Current TOC members:
+
+* Adam Jordens (Netflix)
+* Lars Wander (Google)
+* Rob Zienert (Netflix)
+* Travis Tomsu (Google)
+
+The technical oversight committee is initially seeded with core members from Netflix and Google. We will leave further details such as number of seats, qualifications, tenure, TBD for the time being.
+
+### Steering Committee
+
+The steering committee’s responsibility and function lies in the continual shaping of the governance structure to serve the project’s needs best.
+
+Duties:
+
+* Set governance structure of the project
+* Address questions or concerns from the community about project culture, structure, and clarifications of roles, responsibilities and duties.
+* Ratify new SIGs
+
+Logistics
+
+* [Agenda doc](https://docs.google.com/document/d/1HMdwvBPM4uRFqoeAd7eEkVWIC8dQP40zFavOE5Kq-Eg/edit)
+* Email: [sc@spinnaker.io](mailto:sc@spinnaker.io)
+
+Current SC members:
```

Comment:
You've got a colon after this one, but not after any of the others on this page. What about using heading-level 4 for all.